install_debian_autopsy_4.19.2.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
Use `set -euo pipefail` to make bash fail fast.

Reply:
In this case that would prevent the script from running as the `dpkg -i` commands will fail.

Reply:
Actually, it's possible to use apt which doesn't have this issue.

Reply:
apt can also fail on update -- e.g. database lock file exists.

Reply:
Yes, but apt  doesn't fail when the process goes as intended, while `dpkg -i` will fail even when it runs as intended, as it won't be able to install the dependencies, which requires an additional call to apt. The reason I was using `dpkg -i` is that apt with a local .deb file gives a message that it is selecting a package `sleuthkit-java` and I misunderstood and assumed that it was selecting the debian server package instead of the local .deb file, but looking at the version installed it is installing the .deb file.